CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -0,0 +1,99 @@
+# Release notes for 1.10.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 1.9.0
+
+## Changes by Kind
+
+### Feature
+
+- Add Modify Volume Support for VolumeAttributesClass, this requires:
+  1. The feature gate VolumeAttributesClass and API to be enabled in Kubernetes cluster
+  2. The feature gate VolumeAttributesClass feature gate enabled in external-resizer ([#351](https://github.com/kubernetes-csi/external-resizer/pull/351), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
```

Comment:
While looking at this PR, I noticed that the README needs to be updated:
https://github.com/kubernetes-csi/external-resizer/pull/351/files#r1468258087